pkg/operator/ceph/file/filesystem.go

```diff
@@ -176,6 +169,9 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 					fmt.Sprintf(". %v", err),
 			)
 		}
+		if err := client.SetPoolSize(context, clusterName, f.Name, f.dataPools, f.metadataPool); err != nil {
+			logger.Errorf("failed to set pool size. %v", err)
```

Comment:
@travisn @leseb do you think this should be a logged error or if this should return an error from the function? I'm on the fence. This could mean that the user's data is not as safe as they wish it to be if the pool size is increasing, or it could be less impactful if the size is decreasing. How do we expect that users will see logged error messages that don't cause overt errors? Do we assume they use a log monitor?

Reply:
Agreed that we should return an error here.